docs/enrollRewardProgram.md

```diff
@@ -26,6 +26,9 @@ OPTIONS
 
   -u, --url=url                    [default: http://localhost:3000] the network url
 
+  --leger                          If --ledger is provided, the command will connect and sign the transactions with a
```

Comment:
```suggestion
  --ledger                         If --ledger is provided, the command will connect and sign the transactions using a
```

Reply:
Thanks, this is an internal PR. I'm just sharing code with moha. Thanks for the fixes though...